warehouse/utils/utils.go

```diff
@@ -139,11 +170,22 @@ var DiscardsSchema = map[string]string{
 	"uuid_ts":      "datetime",
 }
 
+var tableNameLimitMap = map[string]int{
+	AZURE_SYNAPSE: 127,
+	BQ:            127,
+	DELTALAKE:     127,
+	MSSQL:         127,
+	POSTGRES:      63,
+	RS:            127,
+	SNOWFLAKE:     127,
+}
```

Comment:
Ideally we want those values encapsulated on their packages

Reply:
Good point.

Reply:
Done.